pom.xml

```diff
@@ -1803,6 +1809,16 @@
                 <artifactId>che-selenium-test</artifactId>
                 <version>${che.version}</version>
             </dependency>
+            <dependency>
+                <groupId>org.keycloak</groupId>
+                <artifactId>keycloak-common</artifactId>
```

Comment:
remove

Reply:
I didn't see how i can remove it now. Did you ?